gpu/gpu_info_cpu.c

```diff
@@ -6,11 +6,12 @@
 void cpu_check_ram(mem_info_t *resp) {
   resp->err = NULL;
   MEMORYSTATUSEX info;
+  info.dwLength = sizeof(info);
```

Comment:
fwiw, this was the *actual* fix.  The rest was just cleanups to get to a point where I could figure out this was the problem

Reply:
Just to verify my understanding here. Setting dwLength here allows `info.ullTotalPhys` and `info.ullAvailPhys` to be calculated correctly by the call to `GlobalMemoryStatusEx` which takes the pointer to the info?

Reply:
Correct.  I didn't look closely enough at the API docs and realize it needs to know the size of the input struct to operate properly.